Description: Add xf86SlowBcopy to all systems
 The `vesa` driver is pretty useless, besides from HURD.
 But, the `xf86SlowBcopy` function is required for it's use.
 Weirdly, this is not included for all systems, even though
 the file seems system independent.
Author: NexusSfan <nexussfan@duck.com>

---

--- a/hw/xfree86/os-support/meson.build
+++ b/hw/xfree86/os-support/meson.build
@@ -155,6 +155,7 @@ else
         'stub/stub_bell.c',
         'stub/stub_init.c',
         'stub/stub_video.c',
+        'misc/SlowBcopy.c',
     ]
 endif
 
